castle/cms/browser/controlpanel/announcements.py

```diff
@@ -44,6 +44,8 @@ class AnnouncementsControlPanelForm(controlpanel.RegistryEditForm):
     def updateFields(self):
         super(AnnouncementsControlPanelForm, self).updateFields()
         self.fields['site_announcement'].widgetFactory = TinyMCETextFieldWidget
+        self.fields['site_disclaimer'].widgetFactory = TinyMCETextFieldWidget
+
```

Comment:
max 2 black lines between class definitions -- I recommend making sure flake8 is setup and integrated into your editor of choice: http://flake8.pycqa.org/en/latest/